shader_lib/flutter/runtime_effect.glsl

```diff
@@ -0,0 +1,20 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
```

Comment:
This seems like an implementation detail of the engine/impellerc, so we should probably package it with the engine artifacts.

Reply:
Made a recipe to include the shader lib here: https://flutter-review.googlesource.com/c/recipes/+/35221

I'm thinking maybe we can have a convention where the "flutter" dir is for stable end-user headers only, and "impeller" is for internals that the stable lib may use with no stability guarantees.

Reply:
Done.